libs/SmartStore/SmartStoreTests/SFSmartStoreTests.m

```diff
@@ -866,38 +866,6 @@ - (void)testQuerySpecPageSize
     XCTAssertEqual(querySpecPageSize, expectedPageSize, @"Page size value should reflect input value.");
 }
 
-#pragma clang diagnostic push
```

Comment:
Do we still need those pragma's?

Reply:
I don't think so. They're ignoring warnings based on nullability markers (or lack thereof). We should have pretty good coverage of nullability markers now.